CodingChallenges/SimplePerceptron/P5/SimplePerceptron/index.html

```diff
@@ -0,0 +1,14 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <title>Perceptron</title>
+    <script src="https://cdn.jsdelivr.net/npm/p5@1.1.9/lib/p5.min.js"></script>
+    <script src="Perceptron.js"></script>
```

Comment:
small thing, but I prefer to use lower case filenames? (If that's silly or there's something I'm missing let me know!)

Reply:
Not sure why i did that tbh, thanks for the feedback!